server/jetstream_cluster.go

```diff
@@ -3499,12 +3542,22 @@ func (js *jetStream) processClusterDeleteConsumer(ca *consumerAssignment, isMemb
 		ca.Group.node.Delete()
 	}
 
+	// Cleanup in case account does not exist or node was nil
+	if sacc := s.SystemAccount(); sacc != nil {
```

Comment:
Again this looks out of place here, maybe we should have this to be pedantic to an extreme, but..